app/assess/data.py

```diff
@@ -482,38 +483,36 @@ def submit_comment(
     return response.ok
 
 
-def get_file_url(filename: str, application_id: str):
+def get_file_response(file_name: str, application_id: str):
     """_summary_: Function is set up to retrieve
     files from aws bucket.
     Args:
         filename: Takes an filename
         application_id: Takes an application_id # noqa
     Returns:
-        Returns a presigned url.
+        Returns a response with a file from aws.
     """
 
-    if filename is None:
+    if file_name is None:
         return None
 
-    prefixed_file_name = application_id + "/" + filename
+    prefixed_file_name = application_id + "/" + file_name
 
     s3_client = boto3.client(
         "s3",
         aws_access_key_id=Config.AWS_ACCESS_KEY_ID,
         aws_secret_access_key=Config.AWS_SECRET_ACCESS_KEY,
         region_name=Config.AWS_REGION,
     )
+
     try:
-        response = s3_client.generate_presigned_url(
-            "get_object",
-            Params={
-                "Bucket": Config.AWS_BUCKET_NAME,
-                "Key": prefixed_file_name,
-            },
-            ExpiresIn=3600,
-        )
+        obj = s3_client.get_object(Bucket=Config.AWS_BUCKET_NAME, Key=prefixed_file_name)
 
+        data = obj['Body'].read()
+
+    # Create a Flask response object
+        response = Response(data, mimetype='text/plain', headers={'Content-Disposition': f'attachment;filename={file_name}'})
```

Comment:
Don't think we should be setting the mimetype to 'text/plain` as response won't be plain text necessarily. Can we try removing the mimetype, or using the content type we  get back from AWS?